spec/services/pii/cacher_spec.rb

```diff
@@ -33,6 +33,29 @@
       expect(decrypted_pii[:ssn]).to eq '5678'
       expect(user_session[:decrypted_pii]).to eq decrypted_pii_json
     end
+
+    it 'updates fingerprints when keys are rotated' do
+      old_ssn_signature = profile.ssn_signature
+      old_email_fingerprint = user.email_fingerprint
+      old_encrypted_email = user.encrypted_email
+
+      old_hmac_key = Figaro.env.hmac_fingerprinter_key
+      allow(Figaro.env).to receive(:hmac_fingerprinter_key_queue).and_return(
```

Comment:
do we think any more tests will test key rotation like this? this blurb is identical in another spec, might be worth adding a helper `rotate_hmac_keys` method for something?